pkg/defaultcomponents/defaults.go

```diff
@@ -68,21 +68,24 @@ import (
 func Components() (otelcol.Factories, error) {
 	var errs error
 
-	extensions, err := extension.MakeFactoryMap(
+	var extensionsList = []extension.Factory{
 		awsproxy.NewFactory(),
 		ecsobserver.NewFactory(),
 		healthcheckextension.NewFactory(),
 		pprofextension.NewFactory(),
 		sigv4authextension.NewFactory(),
 		zpagesextension.NewFactory(),
 		ballastextension.NewFactory(),
+	}
+	extensions, err := extension.MakeFactoryMap(
+		extensionsList...,
 	)
```

Comment:
Why is this on multiple lines but not the other instances of the same pattern?

Reply:
fixed